src/index.html

```diff
@@ -17,6 +17,9 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="minute-hand minute-hand--speed-up"></div>
+      <div class="second-hand second-hand--speed-up"></div>
```

Comment:
Also I think it'll be nice idea to make second-hand and minute-hand `stopwatch` elements

example:
`stopwatch__second-hand`

Reply:
Maked second-hand and minute-hand stopwatch elements.